src/main/scala/thurloe/webservice/SwaggerConfigHttpService.scala

```diff
@@ -0,0 +1,71 @@
+package thurloe.webservice
+
+import com.gettyimages.spray.swagger.SwaggerHttpService
```

Comment:
This `SwaggerHttpService` again isn't used. We're not even using its [`final def routes`](https://github.com/gettyimages/spray-swagger/blob/v0.5.1/src/main/scala/com/gettyimages/spray/swagger/SwaggerHttpService.scala#L60) anywhere in the trait, and they were removed from consumer's composed route [here](https://github.com/broadinstitute/thurloe/pull/14/files#diff-9da5137ca5614efd1f2591aac06eb049L26).